src/controls/filePicker/controls/TilesList/TilesList.tsx

```diff
@@ -76,6 +76,8 @@ export class TilesList extends React.Component<ITilesListProps> {
               getItemCountForPage={this._getItemCountForPage}
               getPageHeight={this._getPageHeight}
               onRenderPage={(pageProps: IPageProps, defaultRender?: IRenderFunction<IPageProps>) => this._onRenderPage(pageProps, defaultRender)}
+              renderedWindowsAhead={this.props.items.length}
```

Comment:
Isn't that too much of extra windows?
If I have 20 items - we will render extra 40 windows.

Reply:
I am not really sure what these properties are responsible for, but without having them in such values component re-renders, when selecting the last item when there is more than 2 pages and displaying only 1st page. With less value it starts to re-render and randomly scrolling to random position